### PR TITLE
LPD-21792 Unable to find dynamic data mapping structure when upgrading from 7.0

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPP-53336](https://liferay.atlassian.net/browse/LPP-53336)
[LPD-21792](https://liferay.atlassian.net/browse/LPD-21792)

### CODE REVIEW

**Review by linolaoj — CHANGES_REQUESTED** [review]
We usually compare structureVersionId1 with structureVersionId2, I suggest either changing the names so we can have structureVersionId1 and structureVersionId2, or changing the name of structureVersionId1.

**Comment by linolaoj** [comment]
Hi @vngoliveira , did you check if LPD-18203 is still good after this fix is applied?

**Comment by vngoliveira** [comment]
@linolaoj, I checked the fix for LPD-18203 and all is working for both issues :smile:

**Comment by linolaoj** [comment]
ci:test:relevant

**Review by linolaoj — APPROVED** [review]

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:relevant

**Comment by vngoliveira** [comment]
ci:test:upgrade

**Comment by linolaoj** [comment]
ci:forward

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_1_0/DDMFieldUpgradeProcess.java [hunk 1]
```diff
@@ -475,13 +475,33 @@ private DDMForm _getFullHierarchyDDMForm(
...
+									"DDMStructureVersion.structureVersionId ",
+									"from DDMStructureVersion WHERE ",
+									"structureId = ?"));
+
+						preparedStatement1.setLong(1, parentStructureId);
+
+						try (ResultSet resultSet1 =
+								preparedStatement1.executeQuery()) {
+
+							if (resultSet.next()) {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_1_0/DDMFieldUpgradeProcess.java:491]: @vngoliveira shouldn't this be `if (resultSet1.next()) {`?

**Pull request closed without merging**